src/forms/FormBuilder.js

```diff
@@ -127,10 +126,11 @@ export default class FormBuilder extends Component {
   renderPreview() {
     if(!this.props.preview) return null;
 
-    const form = Object.assign({}, this.props.forms.form);
```

Comment:
definitely. line 130 `{...form}` does a copy of all the properties into the new `previewForm` object. It's a shorthand `Object.assign`